client/fresheyes-web/src/app/assets/icons/exitArrow.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
```suggestion
"use client"

import React from "react";
```
we should explicitly make this a client component